scrapers/PegasProductions.yml

```diff
@@ -1,21 +1,90 @@
 name: PegasProductions
+
+# This studio has two formats for scenes depending on URL.  The less useful format has '/abonnements' to 
+# start the path  while the more useful one does not.  Scenes appear to be available in both formats, but 
+# there is no map between the resources referenced in each style of URL.
+
 sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pegasproductions.com/abonnements
+    scraper: abonnementsnScraper
+
   - action: scrapeXPath
     url:
       - pegasproductions.com
     scraper: sceneScraper
+
 xPathScrapers:
+  abonnementScraper:
+    scene:
+      Title: //meta[@itemprop="name"]/@content
+      Date:
+        selector: //meta[@itemprop="uploadDate"]/@content
+        postProcess:
+          - replace:
+              - regex: ^([^T]+).+
+                with: $1
+      Performers:
+        Name:
+          selector: //h1
+          postProcess:
+            - replace:
+                - regex: ^.+?([^:]*)$
+                  with: $1
+                - regex: '[^a-zA-Z0-9\s]'
+                  with: '' 
+      Image: //meta[@itemprop="thumbnailUrl"]/@content
+      Studio:
+        Name:
+          fixed: Pegas Productions
+
   sceneScraper:
     scene:
-      # To access other scenes go to Scene and select model which will show all their scenes
       Title:
         selector: //span[@itemprop="name"]
+      
+      # This format has options for English and French language versions of the scene.  Depending on 
+      # the language the values we key off for Performers and Director will be in English or French.
+      Director: //p[contains(b,"Director") or contains(b,"Réalisateur")]/text()
+      Performers:
+        Name: //div[@class="span10"]/p[contains(b,"Starring") or contains(b,"Distribution")]/a
+
       Date:
         selector: //div[@id="date-duree"]/div[1]/p[1]
+        # English URLs display dates in the format of 02/01/2006.  
+        # French URLs display dates in the format of 01 janvier 2006, using the full french name of the month.
+        # This bit of hackery converts the dates used in the French version to format used on the English page.
         postProcess:
+          - replace:
+              - regex: janvier
+                with: 01
+              - regex: f[eé]vrier 
+                with: 02
+              - regex: mars
+                with: 03
+              - regex: avril
+                with: 04
+              - regex: mai
+                with: 05
+              - regex: juin
+                with: 06
+              - regex: juillet
+                with: 07
+              - regex: aout
+                with: 08
+              - regex: septembre
+                with: 09
+              - regex: octobre
+                with: 10
+              - regex: novembre
+                with: 11
+              - regex: d[eé]cembre
+                with: 12
+              - regex: \s
+                with: /
           - parseDate: 02/01/2006
```

Comment:
This fails on the first 9 months of the year because yaml is a wack-ass format: `03` will be turned into just `3` unless you put it in quotes :)

```
DEBU[2024-03-29 14:25:40] Replace: 'mars' with '3'
DEBU[2024-03-29 14:25:40] Before: 26 mars 2024
DEBU[2024-03-29 14:25:40] After: 26 3 2024
<snip>
DEBU[2024-03-29 14:25:40] Before: 26 3 2024
DEBU[2024-03-29 14:25:40] After: 26/3/2024
Error parsing date string '26/3/2024' using format '02/01/2006': parsing time "26/3/2024" as "02/01/2006": cannot parse "3/2024" as "01"
```

I don't know which editor you prefer to use, but I can recommend VS Code with the [YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) from RedHat: it'll warn you about these common pitfalls PLUS it uses the schema file from the repository so you get Stash scraper-specific type hints and autocompletions 🙂